pkg/protoconv/resources/resources.go

```diff
@@ -237,8 +241,11 @@ func (w *DeploymentWrap) populateFields(obj interface{}) {
 		// types do. So, we need to directly access the Pod's Spec field,
 		// instead of looking for it inside a PodTemplate.
 		podSpec = o.Spec
+	// batch/v1beta1 CronJob is deprecated in v1.21+, unavailable in v1.25+.
 	case *batchV1beta1.CronJob:
 		podSpec = o.Spec.JobTemplate.Spec.Template.Spec
+	case *batchV1.CronJob:
```

Comment:
since these are the same, should we consider doing

```
case *batchV1beta1.CronJob, *batchV1.CronJob:
        podSpec = o.Spec.JobTemplate.Spec.Template.Spec
```

instead?

Reply:
No, Go needs to know the exact type that is being upon so case list does not work. I tried.